docs/modules/ROOT/pages/extensions/rest.adoc

```diff
@@ -27,3 +27,8 @@ Please refer to the above links for usage and configuration details.
 ----
 
 Check the xref:user-guide/index.adoc[User guide] for more information about writing Camel Quarkus applications.
+
+== Additional Camel Quarkus configuration
+
+By default this extension configures the xref:extensions/platform-http.adoc as the component that provides the REST transport.
```

Comment:
"By default this extension configures" awakes the impression that the end user can change the default. I guess the user can change it by calling `context.getRestConfiguration().setComponent(...)` from his RoutesBuilder but there is no way to remove platform-http from the class path. I think it would be useful to inform the end user about that, e.g.:

---8<-----

You can override the default from your `RouteBuilder`:

```
import org.apache.camel.builder.RouteBuilder;

public class MyRouteBuilder extends RouteBuilder {
    @Override
    public void configure() throws Exception {
        getContext().getRestConfiguration().setComponent(...);
    }
}
```

Note that by changing the default, the `platform-http` extension is not removed from your application's class path.

---8<-----